mne_bids/read.py

```diff
@@ -385,14 +379,11 @@ def read_raw_bids(bids_basename, bids_root, kind=None, extra_params=None,
 
     # Try to find an associated electrodes.tsv and coordsystem.json
     # to get information about the status and type of present channels
-    search_modifier = f'acq-{acq}' if acq else ''
-    elec_suffix = f'{search_modifier}*_electrodes.tsv'
-    coord_suffix = f'{search_modifier}*_coordsystem.json'
     electrodes_fname = _find_matching_sidecar(bids_fname, bids_root,
-                                              suffix=elec_suffix,
+                                              suffix='electrodes.tsv',
```

Comment:
can you also change the API of the private functions to avoid dealing with legacy API

Reply:
Done!